src/viewer.js

```diff
@@ -2904,6 +2909,10 @@ function onCanvasClick( event ) {
             );
             this.viewport.applyConstraints();
         }
+        else if( gestureSettings.dblClickToZoom ) {
```

Comment:
I think we need a new gesture setting for this new behavior, rather than adding new behavior to the existing gesture setting. I propose `dblClickDragToZoom`. 

Also, the fact that this is an `else if` is why it's not working on desktop, since `clickToZoom` is on by default on desktop. That said, it does bring up an interesting question: What should the behavior be when we have both clickToZoom and dblClickDragToZoom turned on? It looks like right now, if we have clickToZoom and dblClickToZoom turned on, it does them both, so I figure we should continue with that. There's a note in the doc comment for dblClickToZoom that you probably want to turn off clickToZoom; we should have the same note in the doc comment for dblClickDragToZoom.

Reply:
Hi, Thanks a lot for all the efforts & looking into the details!!!   
dblClickZoom & dblClickDragToZoom should work both together. When the user releases the finger after dblClick it does the defined zoom. When the user does not release the finger and starts dragging up/down it does the defined zoom as well.

Reply:
Yeah, that seems perfect!

Reply:
Okay so a seperate setting exists called dblClickDragToZoom. I will add a warning to disable clickToZoom if dblClickDragToZoom needs to be used.

Is that okay ?

Reply:
Sounds good!